doks-cluster.tf

```
@@ -52,3 +52,83 @@ resource "digitalocean_kubernetes_node_pool" "autoscaled-pool" {
     ]
   }
 }
+
+# Data source required as per https://registry.terraform.io/providers/digitalocean/digitalocean/latest/docs/resources/kubernetes_cluster#kubernetes-terraform-provider-example
+# To configure the kuberenetes provider
+data "digitalocean_kubernetes_cluster" "doks" {
+  name       = local.cluster_name
+  depends_on = [digitalocean_kubernetes_cluster.doks_cluster]
+}
+provider "kubernetes" {
+  alias                  = "doks"
+  host                   = data.digitalocean_kubernetes_cluster.doks.kube_config.0.host
+  cluster_ca_certificate = base64decode(data.digitalocean_kubernetes_cluster.doks.kube_config.0.cluster_ca_certificate)
+  # Bootstrap requires to use the Digital Ocean API user as no service account or technical user are created in the cluster
```

```suggestion
# Bootstrap required to use the DigitalOcean API user as no service account or technical user are created in the cluster
```
(nit)

---

```suggestion
  # Bootstrap requires to use the DigitalOcean API user as no service account or technical user are created in the cluster
```

- Indentation must stay: the comment targets the `exec` instruction so it has the same indentation
- Present is expected, it's really important. Using past tense changes the meaning